### PR TITLE
feat: [PIE-4178]: cypress functional test cases for compare YAMl, refactoring

### PR DESCRIPTION
**Pull request by udayvunnam**
##### Summary:

- cypress functional cases for compare YAML, 
- refactoring
- yaml -> YAML text casing

### CODE REVIEW

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Comment by udayvunnam** [comment]
retrigger titlecheck

File: src/modules/70-pipeline/strings/strings.en.yaml [hunk 1]
```diff
@@ -418,8 +418,8 @@ execution:
     pauseStage: Pause Stage
     abortStage: Abort Stage
     downloadLogs: Download logs
-    viewCompiledYaml: View Compiled Yaml
-    compareYamls: Compare Yaml’s
+    viewCompiledYaml: View Compiled YAML
+    compareYamls: Compare YAML’s
```
Comment (chetanvekariya-harness) [inline src/modules/70-pipeline/strings/strings.en.yaml:422]: Is apostrophe intended here ? It should be `Compare YAMLs`, I guess.

File: src/modules/70-pipeline/strings/strings.en.yaml [hunk 2]
```diff
@@ -395,7 +395,7 @@ execution:
   currentExecution: Current Execution
   timePrefix: 'Execution Time: '
   compareAction: Compare
-  compareExecutionsTitle: Select executions to compare the compiled Yaml’s
+  compareExecutionsTitle: Select executions to compare the compiled YAML’s
```
Comment (chetanvekariya-harness) [inline src/modules/70-pipeline/strings/strings.en.yaml:398]: Is apostrophe intended here ? It should be Compare YAMLs, I guess.

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/70-pipeline/strings/strings.en.yaml (hunk 2 above)
Comment (udayvunnam) [reply src/modules/70-pipeline/strings/strings.en.yaml:398]: true, let me check with designer once as this was expected as per design

**Review by ishantmehtaharness — APPROVED** [review]
Looks good

**Pull request merged**